cmake/redisearch_cflags.cmake

```diff
@@ -15,6 +15,10 @@ SET(RS_COMMON_FLAGS "${RS_COMMON_FLAGS} -fPIC -Werror=implicit-function-declarat
 SET(RS_COMMON_FLAGS "${RS_COMMON_FLAGS} -pthread")
 SET(RS_COMMON_FLAGS "${RS_COMMON_FLAGS} -fno-strict-aliasing")
 
+IF (PROFILE)
+    SET(RS_COMMON_FLAGS "${RS_COMMON_FLAGS} -g -fno-omit-frame-pointer")
```

Comment:
Can we trigger a `PROFILE` flag on cmake (i.e. `-DPROFILE=1` or something) and make the resolution in `cmake.debug`?